ci_config.json

```diff
@@ -514,6 +514,17 @@
       "windows": false
     }
   },
+  "libcap": {
+    "_comment": "- relies on Linux specific APIs",
+    "build_on": {
+      "darwin": false,
+      "msys2": false,
+      "windows": false
```

Comment:
These want to see a corresponding `error()` call from the Meson code, whose text matches this test:

```python
if 'unsupported' in error or 'not supported' in error or 'does not support' in error:
```